modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -167,3 +169,25 @@ main `react` package.
 - `Box` was promoted from `@workday/canvas-kit-labs/common` to `@workday/canvas-kit-react/layout`
 - `Flex` was promoted from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
 - `Stack` was promoted from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
+
+## Tokens Updates
+
+### Depth Tokens
+
+**Changes:**
+
+- Depth `inset` was removed from token list.
+- Added new depth: `none` (to remove default depth from component), `5` and `6`.
+- Now all depth will add only `box-shadow` with 2 shadows style and no border.
+
+Also we changed default depth for the next components:
+
+- Card: Depth 2 → Depth 1
+- Select Menu, Menu: Depth 2 → Depth 3
```

Comment:
Q: Should "Select Menu" and "Menu" be on separate lines?

Reply:
There's also a Select component in the main package. Maybe we should say "Preview Select Menu" to be more clear?